component/mcsdadmin/component.go

```diff
@@ -443,26 +447,38 @@ func newLocationPost(w http.ResponseWriter, r *http.Request) {
 	renderList[fhir.Location, tmpls.LocationListProps](client, w, tmpls.MakeLocationListXsProps)
 }
 
-func listLocations(w http.ResponseWriter, r *http.Request) {
+func listLocations(w http.ResponseWriter, _ *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	renderList[fhir.Location, tmpls.LocationListProps](client, w, tmpls.MakeLocationListXsProps)
 }
 
-func homePage(w http.ResponseWriter, r *http.Request) {
+func homePage(w http.ResponseWriter, _ *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	tmpls.RenderWithBase(w, "home.html", nil)
 }
 
-func notImplemented(w http.ResponseWriter, r *http.Request) {
-	w.WriteHeader(http.StatusNotImplemented)
-	tmpls.RenderWithBase(w, "home.html", nil)
-}
-
-func notFound(w http.ResponseWriter, r *http.Request) {
+func notFound(w http.ResponseWriter, _ *http.Request) {
 	w.WriteHeader(http.StatusNotFound)
 	_, _ = w.Write([]byte("Path not implemented"))
 }
 
+func deleteHandler(w http.ResponseWriter, r *http.Request) {
+	orgId := r.PathValue("id")
+	pathResType := r.PathValue("resourceType")
+	resourceType := cases.Title(language.English, cases.Compact).String(pathResType)
```

Comment:
Lol, does this work? Maybe we want a unit test for this (feel free to keep out of scope though)

And now I'm thinking... maybe we shouldn't do this after all, since it doesn't limit you from deleting other than the supported resources